### PR TITLE
openstack: install, enable and start fail2ban on teuthology VM

### PR DESCRIPTION
**Pull request by jan--f**
Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
@jan--f Please rebase on top of latest master.

File: teuthology/openstack/setup-openstack.sh [hunk 1]
```diff
@@ -117,6 +117,17 @@ function setup_docker() {
...
+    if test -f /usr/bin/fail2ban-server; then
+        echo "OK fail2ban is installed"
+    else
+        sudo apt-get install fail2ban
+        echo "INSTALLED fail2ban"
+    sudo systemctl restart fail2ban
+    sudo systemctl enable fail2ban
+    echo "STARTED fail2ban"
+}
+
```
Comment (smithfarm) [inline teuthology/openstack/setup-openstack.sh]: how is this function going to get called?

Comment (jan--f) [inline teuthology/openstack/setup-openstack.sh]: oh well it wasn't I guess :) fixed

**Comment by jan--f** [comment]
rebased

**Comment by smithfarm** [comment]
@dachary Ping

**Comment by smithfarm** [comment]
@jan--f What testing have you done on this?

**Comment by jan--f** [comment]
@smithfarm only manual so far. I.e. installed it in a teuthology VM. So far no bans observed though.

**Comment by jan--f** [comment]
ok, successfully tested. VM is up here http://158.69.76.107:8081/

Fail2ban is installed and started:
```
(virtualenv) ubuntu@teuthology:~$ systemctl status fail2ban.service 
● fail2ban.service - Fail2Ban Service
   Loaded: loaded (/lib/systemd/system/fail2ban.service; enabled; vendor preset: enabled)
   Active: active (running) since Tue 2017-03-21 08:13:26 UTC; 3min 40s ago
     Docs: man:fail2ban(1)
 Main PID: 32062 (fail2ban-server)
    Tasks: 3
   Memory: 9.3M
      CPU: 600ms
   CGroup: /system.slice/fail2ban.service
           └─32062 /usr/bin/python3 /usr/bin/fail2ban-server -s /var/run/fail2ban/fail2ban.sock -p /var/run/fail2ban/fail2ban.pid -x -b

Mar 21 08:13:25 teuthology systemd[1]: Stopped Fail2Ban Service.
Mar 21 08:13:25 teuthology systemd[1]: Starting Fail2Ban Service...
Mar 21 08:13:25 teuthology fail2ban-client[32056]: 2017-03-21 08:13:25,983 fail2ban.server         [32060]: INFO    Starting Fail2ban v0.9.3
Mar 21 08:13:25 teuthology fail2ban-client[32056]: 2017-03-21 08:13:25,984 fail2ban.server         [32060]: INFO    Starting in daemon mode
Mar 21 08:13:26 teuthology systemd[1]: Started Fail2Ban Service.
```

**Pull request merged**